k8s/prod/common/family-public-law/fpl-case-service.yaml

```diff
@@ -34,6 +34,7 @@ spec:
         DOCMOSIS_TORNADO_URL: https://docmosis.platform.hmcts.net
         IDAM_CLIENT_ID: fpl_case_service
         IDAM_CLIENT_REDIRECT_URI: https://fpl-case-service-prod.service.core-compute-prod.internal/oauth2/callback
+        GATEWAY_URL: http://gateway-ccd.prod.platform.hmcts.net
```

Comment:
This is wrong. Where did you take it from. It should be `https://gateway.ccd.platform.hmcts.net`.

Reply:
thanks for spotting this. Was in `prod.tfvars`. Now updated